components/script/dom/audiotracklist.rs

```diff
@@ -70,29 +70,24 @@ impl AudioTrackList {
     }
 
     pub fn set_enabled(&self, idx: usize, value: bool) {
-        let track = match self.item(idx) {
-            Some(t) => t,
-            None => return,
-        };
-
-        // If the chosen tracks enabled status is the same as the new status, return early.
+        let track = self.item(idx).unwrap_or_else(|| return);
+    
         if track.enabled() == value {
             return;
         }
-        // Set the tracks enabled status.
+    
         track.set_enabled(value);
-        if let Some(media_element) = self.media_element.as_ref() {
+        if let Some(media_element) = &self.media_element {
             media_element.set_audio_track(idx, value);
         }
-
-        // Queue a task to fire an event named change.
+    
         let global = &self.global();
         let this = Trusted::new(self);
         let (source, canceller) = global
             .as_window()
             .task_manager()
             .media_element_task_source_with_canceller();
-
+    
```

Comment:
You have added whitespace here. You need to call `./mach fmt` before uploading PRs. This will also cause `./mach test-tidy` to fail.